source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -164,6 +201,7 @@ OpenGLContext: class extends GpuContext {
 		target canvas draw(source, map)
 	}
 	createFence: override func -> GpuFence { OpenGLFence new(this) }
+	toRasterAsync: override func (source: GpuImage) -> Future<RasterImage> { _DummyFuture new(this toRaster(source)) }
```

Comment:
The default implementation will actually be synchronized, so we wait on a dummy future.